src/elements/Select.tsx

```diff
@@ -155,8 +152,8 @@ const SmallSelectContainer = styled.div.attrs<SelectProps>({})`
     cursor: pointer;
     pointer-events: none;
     position: absolute;
-    top: 12px;
-    margin-left: 8px;
+    top: 10px;
+    margin-left: -8px;
```

Comment:
this damn thing 🤕 The whole select component could probably be rewritten... Not my best code.

Reply:
All good, we do what we can with the time we have ;)
I'll take a stab at cleaning things up a bit.

Reply:
It's actually not bad at all, we just gotta account for the arrow, that's the only issue with it.